src/Module/Security/OpenID.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Friendica\Module\Security;
+
+use Friendica\App\Authentication;
+use Friendica\App\BaseURL;
+use Friendica\BaseModule;
+use Friendica\Core\Config\Configuration;
+use Friendica\Core\L10n\L10n;
+use Friendica\Core\Session\ISession;
+use Friendica\Database\Database;
+use Friendica\Util\Strings;
+use LightOpenID;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Performs an login with OpenID
+ */
+class OpenID extends BaseModule
+{
+	public static function init(array $parameters = [])
```

Comment:
This is unnecessary since there isn't a `post()` method the `init()` method could be shared with.